src/styles/CommitteesSection.module.css

```
@@ -0,0 +1,71 @@
+.container {
+    padding-bottom: 5rem;
+}
+
+.content {
+    display: flex;
+    width: 100%;
+}
+
+.description {
+    width: 50%;
+    flex: 1;
+}
+
+.descriptionLeft {
+    font-size: 16px;
+    font-weight: 400;
+    color: #21636C;
```

```suggestion
    font-weight: 500;
```
This should make the text more readable